tests/EventListener/ConditionallyRemoveVaryHeaderListenerTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\Tests\EventListener;
```

Comment:
This should be in new namespace as that ease of process of merging things up

Reply:
Fixed in 5ef18198be02212b17bafdd20f36adbc804188ba and 58e47345fbb72a9230c2bc675e8e3d4da5244736

After changing copyuright to Ibexa too, Checker complains about `Warning: Found violation(s) of type: header_comment`..
It should remain eZ Systems?
please doublecheck I get got the namespace right

Reply:
It should remain as eZ Systems for older branches. It is necessary to change it when merging up, as for newer ones codestyle fixer will expect Ibexa.

Reply:
Okay, I reverted the change in the copyright comment